dev/Generated/NumberBox.properties.cpp

```diff
@@ -226,7 +226,7 @@ void NumberBoxProperties::EnsureProperties()
                 winrt::name_of<winrt::TextAlignment>(),
                 winrt::name_of<winrt::NumberBox>(),
                 false /* isAttached */,
-                ValueHelper<winrt::TextAlignment>::BoxValueIfNecessary(winrt::TextAlignment::Left),
```

Comment:
Why did you remove the default value? TextBox.TextAlignment/HorizontalTextAlignment uses TextAlignment::Left as their default value and we should pick the same default here.

To add a default value, you add the default value in the NumberBox .idl file by adding `[MUX_DEFAULT_VALUE("winrt::TextAlignment::Left")]` above the TextAlignment entry (you can find examples in the .idl file).

Reply:
Agree with @Felix-Dev here, please set the default value to `TextAlignment::Left`.

Reply:
Don't know what happened with the Default. It is now back to Left. Sorry...

Reply:
@sonnemaf It appears to me you have manually edited the NumberBox.properties.cpp file to include the default value in commit https://github.com/microsoft/microsoft-ui-xaml/pull/2901/commits/dc9cef730e98f749bc7148b8f4cd44bec22de93d. Since that file is auto-generated (for example whenever a new API is added to the NumberBox.idl file) manually added code *will not* persist here. In fact, I wouldn't be surprised if that was the reason the default value was removed in a previous commit (the file was probably automatically generated again and you committed that new version). 

Instead, to specify a persisting default value for a public API, please add it to the .idl file like so:
```
[WUXC_VERSION_PREVIEW]
{
      [MUX_DEFAULT_VALUE("winrt::TextAlignment::Left")]
      Windows.UI.Xaml.TextAlignment TextAlignment;
}
```

This will generate the correct code whenever a new version of that file is automatically created.

Reply:
Yes I was editing the NumberBox.properties.cpp manually. I fixed it now using the MUX_DEFAULT_VALUE attribute in the .idl file. C++ is totally new for me. Thanks for your help.